### ISSUE TITLE
With 'dark mode' enabled on a Macbook, some colors make text hard to read

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#882]
User (Jason Womack) request: 
I currently use a Mac with my OS setting set for Dark Mode theme.  I noticed that the search and filter elements seem to pick up that setting and react their UI to that value.  When that happens the UI looks off and the column selector is not very usable due to is font have lack of contrast.  See screenshots as examples.

<img width="1824" height="1006" alt="Image" src="https://github.com/user-attachments/assets/9a2b5daf-15ca-490a-bfb2-f0039c03953e" />

<img width="1820" height="1006" alt="Image" src="https://github.com/user-attachments/assets/78eb30ca-6230-4df2-89d5-f8cc0e330f44" />
